dockerfiles/contracts-ci-linux/Dockerfile

```diff
@@ -39,12 +39,12 @@ RUN set -eux; \
 # use minimum components
 	rustup set profile minimal; \
 # install nightly Rust
-	rustup install nightly; \
+	rustup install nightly-2020-10-01; \
```

Comment:
Hm I think we might need to add `rustup default nightly-2020-10-01`.

Failure: https://gitlab.parity.io/parity/cargo-contract/-/jobs/691564#L55

Reply:
It would be overridden by [RUSTUP_TOOLCHAIN](https://github.com/paritytech/cargo-contract/blob/b3913161342eae2f631de5d437eecfe55fea3ac4/.gitlab-ci.yml#L20), should remove that line as well.